src/SFA.DAS.DATA.MANAGEMENT/Views/Comp_DAS_Compliance_Report .sql

```diff
@@ -0,0 +1,159 @@
+﻿CREATE VIEW [Comp].[DAS_Compliance_Report]
+	AS 
+
+SELECT			
+	CN.DASAccountID		
+	,ISNULL(CAST(Acc.Name as nvarchar(100)),'NA') as DASAccountName	
+	,ISNULL(LD.PayrollYear, '2999') AS PayrollYear
+	,ISNULL(LD.PayrollMonth, '99') AS PayrollMonth		
+	,CN.CalendarYear		
+	,CN.CalendarMonth		
+	,SUM(ISNULL(LevyDeclaredInMonth, 0)) AS LevyDeclaredInMonth		
+	,SUM(ISNULL(PaymentLevy, 0)) AS PaymentLevy		
+	,SUM(ISNULL(PaymentCoInvestedESFA, 0)) AS PaymentCoInvestedESFA		
+FROM			
+(
+	SELECT DISTINCT		
+		 CASE	
+		 	WHEN LD.PayrollMonth BETWEEN 1 AND 8 THEN LD.PayrollMonth + 4
+		 	ELSE LD.PayrollMonth - 8
+		 END AS CalendarMonth -- align	
+		,CASE	
+			WHEN LD.PayrollMonth BETWEEN 1 AND 8 THEN CONCAT('20', LEFT(LD.PayrollYear, 2))
+			ELSE CONCAT('20', FORMAT(CONVERT(INT, SUBSTRING(LD.PayrollYear, 4, 2)), '00'))
+		END AS CalendarYear
+		 ,LD.PayrollMonth
+		 ,LD.PayrollYear
+		 ,ISNULL(EA.HashedId,'XXXXXX') AS DASAccountID
+	FROM AsData_PL.Acc_Account EA
+	LEFT JOIN ASData_PL.Fin_GetLevyDeclarationAndTopUp AS LD
+		ON EA.ID=LD.AccountId
+	WHERE LD.LastSubmission=1
+	UNION		
+	SELECT DISTINCT
+		cp.CalendarMonth
+		,
+		cp.CalendarYear
+		,
+		CASE	
+			WHEN cp.CalendarMonth BETWEEN 1 AND 4 THEN cp.CalendarMonth + 8
+			ELSE cp.CalendarMonth - 4
+		END AS PayrollMonth	
+		,
+		CASE	
+			WHEN cp.CalendarMonth BETWEEN 1 AND 4 THEN CONCAT(CAST(RIGHT(cp.CalendarYear, 2) -1 AS VARCHAR), '-', CAST(RIGHT(cp.CalendarYear, 2) AS VARCHAR))
+			ELSE CONCAT(CAST(RIGHT(cp.CalendarYear, 2) AS VARCHAR),'-', CAST(RIGHT(cp.CalendarYear, 2) + 1 AS VARCHAR))
+		END AS PayrollYear	
```

Comment:
Same Here, The Calculation seems to be deriving Academic Year/Month and not Payroll Year/Month. Is this Intended?
Alternatively, DASCalendarMonth table can be used to derive Payroll(Tax) Year/Month by joining on Calendar Year & Calendar Month Number.

Reply:
The New Changes looks good to me now.